django/core/cache/backends/memcached.py

```diff
@@ -144,10 +144,15 @@ def decr(self, key, delta=1, version=None):
 
     def set_many(self, data, timeout=DEFAULT_TIMEOUT, version=None):
         safe_data = {}
+        keys_mapping = {}
         for key, value in data.items():
-            key = self.make_key(key, version=version)
-            safe_data[key] = value
-        self._cache.set_multi(safe_data, self.get_backend_timeout(timeout))
+            transformed_key = self.make_key(key, version=version)
+            safe_data[transformed_key] = value
+            keys_mapping[transformed_key] = key
+        failing_keys = self._cache.set_multi(safe_data,
```

Comment:
Use a single line (we allow up to 119 chars when it helps readability) or we use hanging indent.